pkg/version/version_test.go

```diff
@@ -0,0 +1,79 @@
+// Copyright 2022 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package version
+
+import (
+	"fmt"
+	"runtime"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestGetFullVersion(t *testing.T) {
+	gotversion := GetFullVersion()
+	assert.Equal(t, "UNKNOWN", gotversion)
+
+	Version = "1.9"
+	ReleaseStatus = "released"
+	gotversion = GetFullVersion()
+	assert.Equal(t, "1.9", gotversion)
+
+	Version = "1.10"
+	ReleaseStatus = "unreleased"
+	gotversion = GetFullVersion()
+	assert.Equal(t, "1.10-unknown", gotversion)
+
+	Version = "1.10"
+	ReleaseStatus = "unreleased"
+	GitSHA = "abc"
+	GitTreeState = "dirty"
+	gotversion = GetFullVersion()
+	assert.Equal(t, "1.10-abc.dirty", gotversion)
+
+	Version = "1.10"
+	ReleaseStatus = "unreleased"
+	GitSHA = "abc"
```

Comment:
Ditto, I think you can remove duplicate assignment if they are already assigned before. You should just need to assign them when you need to replace existing value.